pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1578,7 +1578,6 @@ class NetworkStandaloneSpoke(StandaloneSpoke):
     uiFile = "spokes/network.glade"
 
     preForHub = SummaryHub
-    priority = 10
```

Comment:
Please, don't change the priorities when it is not necessary. There might be addons that rely on these values.

Reply:
OK